plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven.test/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenArchetypeProjectWizardTest.java

```diff
@@ -142,6 +144,37 @@ public void run() {
         assertEquals("ac", page.javaPackageField.getText());
       }
     });
+  }
 
+  @Test
+  public void testAutoPackageNameSetterOnGroupIdChange_disbledOnUserChange() {
+    Realm.runWithDefault(DisplayRealm.getRealm(Display.getDefault()), new Runnable() {
```

Comment:
Interesting. Perhaps this will make the test work even if `<useUIThread>false<useUIThread>`?